### ISSUE TITLE
cloudflare tunnel instance of jellyfin doesn't work

### ISSUE THREAD

**Issue by r4nchy** [rasmuslos/AmpFin#72]
I am trying to access my jellyfin behind cf tunnel using the Ampfin, but the app doesn't recognize it.
how do we make it possible ??

**Comment by rasmuslos** [comment]
Are you using zero trust?

**Comment by r4nchy** [comment]
I don't know what that is, maybe I was using it.

But I also tried tried accessing it using VPN server but that didn't work either.

The domain works perfectly fine in the web browser

**Comment by rasmuslos** [comment]
I don't fully understand your setup, can you explain in detail how traffic is routed from your phone to your server. Please include things like which DNS server (e.g. if you are using a FritzBox it might block DNS queries to your IP) is used and through which providers it passes.

**Comment by rasmuslos** [comment]
> just a side note, I tried the same network configuration for the shelfplayer, and it seems to detect the abs server instance but the login always fails. I don't know why that is the case.

Its really odd that it can detect the server but logging in fails, but please open a separate issue for that

**Comment by r4nchy** [comment]
![](https://i.imgur.com/Z4BTetY.jpg)

I don't know much about network, still learning but here is my setup. on the left is my homelab. And on the right is my remote clients (ipad) with which I am trying to remotely access homelab.

Today I have two ways to connect to my homelab either using the Wireguard VPN (Case1) or the cloudflared tunneling (Case2)

To the left I have an ISP router and behind that I have another router (Omada) which is the main router I use for all the devices. I run pihole on one of the pi, which acts as a DNS server as well.

**Comment by r4nchy** [comment]
Just to be clear when my client device (ios device) are under the omada, then the login works fine. As I have reverse proxy setup using caddy in a docker along with abs and jf in the same server.

jf.example.org {
    reverse_proxy http://192.168.0.104:7601
    tls {
        dns cloudflare **************************
    }
}

Remotely, I tried different applications. I tried SenPlayer, and the jellyfin login was successful with both Cloudflared tunnel and VPN server connections.
